src/staff-handbook/guidance-on-using-slack.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Guidance on using Slack
+last_reviewed_at: 2024-07-22T12:56:00.454Z
+---
+## Expected behaviour
+
+At dxw we encourage staff to exchange ideas and opinions. We also want everyone to feel safe and heard. Here are some principles we should all follow when communicating online to support an inclusive culture:
+
+* be respectful
+* be kind
+* be constructive in any criticism
+
+Please be mindful that conversations in public channels on Slack are happening in the open and may get shared outside of the company.
+
+## Examples of unacceptable behaviour
+
+* conduct or speech which might be considered sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory or offensive in nature
+* intimidation or harassment
+* disrespect towards differences of opinion
+* inappropriate attention or contact. Be aware of how your actions affect others. If it makes someone uncomfortable, stop
+* not understanding the differences between constructive criticism and disparagement
+* threats of violence or violent language
+
+Slack encourages quick and informal communication, but be mindful that your conversations on Slack represent dxw.
+
+## Upholding these principles
+
+Everyone should feel comfortable holding people to these standards. 
+
+If you feel that someone has breached these guidelines, speak to your line manager or the People team. They will be able to help you find an appropriate resolution.
+
+# Using Slack channels, threads and direct messages (DMs)
```

Comment:
This will need to be a h2 (`##`) since the page title should be the only h1. The headings that follow will need to go down one level each (h2 -> h3 etc)